### PR TITLE
Fixes #22941: Titles are not consistant in the new Operatin System inventory tab with group search

### PR DESCRIPTION
**Pull request by fanf**
https://issues.rudder.io/issues/22941

Make thing identical than in node search / "node summary"

### CODE REVIEW

**Comment by fanf** [comment]
PR updated with a new commit

File: webapp/sources/rudder/rudder-web/src/main/scala/com/normation/rudder/web/services/DisplayNode.scala [hunk 1]
```diff
@@ -759,18 +759,24 @@ object DisplayNode extends Loggable {
...
+          ("Operating System Service Pack", escape(sm.node.main.osDetails.servicePack.getOrElse("None"))),
+          ("Operating System Architecture Description", escape(sm.node.archDescription.getOrElse("None"))),
+          ("Kernel Version", escape(sm.node.main.osDetails.kernelVersion.value)),
+          ("Total Physical Memory (RAM)", escape(sm.node.ram.map(_.toStringMo).getOrElse("-"))),
           ("Manufacturer", escape(sm.machine.flatMap(x => x.manufacturer).map(x => x.name).getOrElse("-"))),
           ("Machine Type", displayMachineType(sm.machine).text),
-          ("Total swap space", escape(sm.node.swap.map(_.toStringMo).getOrElse("-"))),
+          ("Total Swap Space (Swap)", escape(sm.node.swap.map(_.toStringMo).getOrElse("-"))),
           ("System Serial Number", escape(sm.machine.flatMap(x => x.systemSerialNumber).getOrElse("-"))),
+          ("Node State", escape(optNode.map(n => getNodeState(n.state)).getOrElse("-"))),
```
[the remark this answers is not in the document]
Comment (fanf) [reply webapp/sources/rudder/rudder-web/src/main/scala/com/normation/rudder/web/services/DisplayNode.scala:775]: done

**Review by ncharles — APPROVED** [review]

**Comment by fanf** [comment]
PR updated with a new commit

**Comment by fanf** [comment]
OK, squash merging this PR

**Pull request merged**